src/components/Button/Button.tsx

```diff
@@ -181,11 +246,16 @@ const Button = forwardRef(
         "aria-disabled": disabled,
         "aria-labelledby": ariaLabeledBy,
         "aria-label": ariaLabel,
-        "aria-busy": loading ? "true" : undefined,
         "aria-haspopup": ariaHasPopup,
         "aria-expanded": ariaExpanded,
         "aria-controls": ariaControls
       };
+
+      if (loading) {
+        props["aria-busy"] = "true";
+      }
+
+      return props;
```

Comment:
interesting - why did you need to put the `loading` outside of the object?

Reply:
The explicit `undefined` messed TS's type inference when spreading onto the element later on.